flytekit/configuration/internal.py

```diff
@@ -31,6 +31,11 @@ def get_specified_images(cfg: ConfigFile) -> typing.Dict[str, str]:
         return images
 
 
+class Deck(object):
+    SECTION = "deck"
+    ENABLE_DECK = ConfigEntry(LegacyConfigEntry(SECTION, "enable_deck", bool))
```

Comment:
Why do we need to gate this?  Can't we just enable it for everyone?  Or

Reply:
I think it should be possible to disable maybe as a perf thing

Reply:
But let's do it later